vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -212,42 +212,73 @@
       </ComplexType>
 
       <Function Name="isroot" IsBound="true">
-        <Annotation Term="Core.Description" String="Returns true, if and only if the value of the node property of the specified hierarchy is the root of the hierarchy" />
+        <Annotation Term="Core.Description" String="Returns true for a given entity, if and only if the value of the node property of the specified hierarchy annotation is the root of the hierarchy in the given entity set of hierarchy nodes" />
         <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false" />
-        <Parameter Name="Hierarchy" Type="Edm.String" Nullable="false" />
+        <Parameter Name="Nodes" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Name of the entity set holding the nodes of the recursive hierarchy" />
+        </Parameter>
+        <Parameter Name="Hierarchy" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Qualifier to identify the `RecursiveHierarchy` annotation applied to the type of the entity set referenced in `Nodes`" />
+        </Parameter>
         <ReturnType Type="Edm.Boolean" Nullable="false" />
       </Function>
 
       <Function Name="isdescendant" IsBound="true">
-        <Annotation Term="Core.Description" String="Returns true, if and only if the value of the node property of the specified hierarchy is a descendant of the given parent node with a distance of less than or equal to the optionally specified maximum distance" />
+        <Annotation Term="Core.Description" String="Returns true for a given entity, if and only if the value of the node property of the specified hierarchy annotation is a descendant of the given parent node in the given entity set of hierarchy nodes with a distance of less than or equal to the optionally specified maximum distance" />

```

Comment:
Can it depend on the entity set whether a given node is a descendant of another?